wasmer/instance.go

```diff
@@ -280,8 +289,12 @@ func (instance *Instance) Cache() ([]byte, error) {
 	}
 
 	goBytes := C.GoBytes(unsafe.Pointer(cacheBytes), C.int(cacheLen))
-	cacheBytes = nil
 
+	hackMap := make(map[unsafe.Pointer]interface{})
```

Comment:
here too? Why do we need the hack map?

Reply:
testing now - as without it is fails in some scenarios.